package.json

```diff
@@ -74,11 +78,12 @@
   },
   "homepage": "https://github.com/FabricLabs/fabric#readme",
   "dependencies": {
-    "@webassemblyjs/ast": "^1.8.5",
+    "@fabric/zmq": "github:FabricLabs/fabric-zmq#master",
```

Comment:
I'm going to take a second pass at the inclusion of ZMQ here — we may want to offer it as a core service